app/assets/stylesheets/pages/_resources_show.scss

```diff
@@ -65,19 +79,22 @@ margin-top: 40px;
   margin-left: auto;
   margin-right: auto;
 }
+
 iframe {
   height: 600px;
   width: 800px;
-  margin-top: -140px;
+  margin-top: -160px;
   border-radius: 10px;
 }
-.description-pdf-container {
```

Comment:
Is it possible you forgot the "}" in the description-pdf-container?

Reply:
those lines of code are commented though